pkg/odo/cli/config/set.go

```diff
@@ -30,6 +32,11 @@ var (
    %[1]s %[8]s 0.5 
    %[1]s %[9]s 2 
    %[1]s %[10]s 1 
+
+   # Set a env variable in the local config
+   %[1]s %[2]s --env PORT=4000
+   %[1]s %[3]s --env DB_USERNAME=postgres,DB_HOSTNAME=postgres
+   %[1]s %[4]s --env KAFKA_HOST=kafka --env KAFKA_PORT=6639
```

Comment:
This is incorrect...

See below output from help:

```sh
  # Set a env variable in the local config
  odo config set Type --env PORT=4000
  odo config set Name --env DB_USERNAME=postgres,DB_HOSTNAME=postgres
  odo config set MinMemory --env KAFKA_HOST=kafka --env KAFKA_PORT=6639

  # Unset a configuration value in the local config
  odo config unset Type
  odo config unset Name
  odo config unset MinMemory
  odo config unset MaxMemory
  odo config unset Memory
  odo config unset Ignore
  odo config unset MinCPU
  odo config unset MaxCPU
  odo config unset CPU

  # Unset a env variable in the local config
  odo config unset Type --env PORT
  odo config unset Name --env DB_USERNAME,DB_HOSTNAME
  odo config unset MinMemory --env KAFKA_HOST --env KAFKA_PORT
```

Reply:
Do we really need to show all three options? It makes the help look convoluted :-(

Reply:
so what do you suggest @kadel

Reply:
kept only one example

Reply:
resolve the incorrect output too @cdrage